### ISSUE TITLE
dsPlot.m: change Variables' Power_SUA and Power_MUA to Power_SU and Power_MU

### ISSUE THREAD

**Issue by kupiqu** [DynaSim/DynaSim#279]
I think SUA and MUA power are very specific things (A for activity). In particular, they typically refer to power spectrum of high passed filtered LFPs, that is, spikes. MUA without spikes sorting (all spikes together) and SUA with spike sorting (single cells).

Therefore to avoid confussion I would just use in dsPlot.m power_SU and power_MU, so each refers to the power spectrum  of variables for a single unit and multiunit, respectively.

Jason, do you think it makes sense?

**Comment by erik-roberts** [comment]
this would also need to be changed in dsCalcPower and the one for classifying in the GV branch.

**Comment by asoplata** [comment]
I'm probably just being thick, but I still don't understand what, specifically, is the difference between, for example, SU and SUA (and, respectively, MU and MUA)? Is it:

1. That SUA always refers to LFP / a **voltage** signal, and therefore SUA of non-voltage variables isn't a "proper" SUA?
2. That SUA is high-pass filtered instead of just the raw signal?
3. That SUA is just an average of simulated intracellular traces, instead of a "real" electrode recording?
4. Or something else entirely?

Furthermore, are both SU/MU and SUA/MUA used in literature in this same way? I think that whatever we change has to be done consistently and has to make sense to the first degree to anyone using them. 

Relatedly, would it ever make sense to have different functions compute SU vs SUA in DS? Because I think that would be super confusing. I don't care about no-A vs with-A except that with-A seems to be much more readily used in the literature. And anyone who's using DynaSim is aware that they're running simulations, and that the output is NOT real, actual MUA recordings.

**Comment by kupiqu** [comment]
So in summary:

- SU and MU can refer to anything as they mean single unit and multi-unit, so one can say sth like MU voltage trace or MU voltage power, etc.

- "A" in SUA and MUA refers to activity, which as far as I know refers to spiketimes as in a rastergram, or if you want a binary signal which is only 1 at the time spikes are triggered, 0 elsewhere.

**Comment by kupiqu** [comment]
Let me be more explicit wrt to this sentence:

> SUA/MUA comes from the same signal but high-pass filtered to acess just the spikes, which are discreet events in time.

Experimentalists do not only high-pass filter, they also use a threshold to just get the spiketimes.

**Comment by kupiqu** [comment]
In my humble opinion, we should think if we want to use MU voltage traces/power as they are not directly comparable to experimental observations.

- It's an open question but LFP seems to be more related to synaptic inputs than to voltage traces. One can surrogate LFPs with synaptic gatings. Alternatively, low-pass filter voltage traces would remove spikes artifacts and would make it more similar to LFPs

- SUA/MUA analyses just take spike times into consideration

In this way, (mean) voltage traces are sth in between, and of course are correlated to these two, but there is no easy association to any of them...

**Comment by kupiqu** [comment]
Yes, I guess I am biased based on what I encountered before and some experimentalists do differently than others to this regard. In any case, I agree that averaged vs individual voltage makes much more sense because it avoids all potential confusion.
